frontend/src/components/RightSideBar/AircraftTable/AircraftTable.vue

```diff
@@ -6,32 +6,46 @@
       <q-btn flat icon="close" size="lg" @click="close" />
     </div>
     <div id="tab-content">
-      <div class="row">
-        <q-card class="map-card bg-secondary text-white" flat square>
-          <q-card-section>
-            <div class="row">ADD YOUR STUFF HERE</div>
-          </q-card-section>
-        </q-card>
-      </div>
+            <q-table
+                            title="Aircrafts"
+                            :data="planes"
+                            :columns="columns"
+                            row-key="name"
+                            hide-bottom
+                            dark
+                    >
+                    </q-table>
     </div>
     <br />
     <br />
   </div>
 </template>
 
 <script>
+import planeStore from "src/planeStore";
 export default {
   components: {},
   setup(props, { emit }) {
     const mapStyles = [];
     const close = () => {
       emit("closeTest");
     };
-
     return {
       close,
     };
   },
+  data () {
```

Comment:
After moving the columns variable remove the data() {return{}}
data function is only for the options api but we are using the composition api.